### PR TITLE
Update The Attack Surface documentation

### PR DESCRIPTION
**Pull request by ojrb**
Hi @mypil this PR solved the issue #1168.  I add the compatibility with Windows 10 version 1703, explain the Engine Version compatibility for Windows 10 and add a reference for the Windows Defender compatibility with other antivirus.  Please review and merge !!

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ Event ID | Description
 1121 | Event when rule fires in Block-mode
 1122 | Event when rule fires in Audit-mode
 
+The engine version of the attack surface reduction events in the event log, is part of the Widnows Defender product, not of the Oeprating System, but because Windows Defender is integrated with Windows 10 Operating System, this feature works on all machines with Windows 10 installled.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md]:
```suggestion
The "engine version" of attack surface reduction events in the event log, is part of  Windows Defender ATP, not of the operating system. Windows Defender ATP is integrated with Windows 10, so this feature works on all machines with Windows 10 installed.
```

**Comment by mypil** [comment]
@ojrb - please check on the suggested copy/edit review changes made by @nenonix and merge the commit. thanks

**Comment by ojrb** [comment]
Ready @mypil committed the @nenonix suggestions !

**Comment by mypil** [comment]
@levinec - Can you please assist to check, approve and merge the content of this PR based on issue #1168?

Thank you.

**Comment by levinec** [comment]
@mypil need to update "Windows Defender ATP" to "Microsoft Defender ATP". Then will approve!

**Comment by ojrb** [comment]
@mypil - Ready the changes proposed by @levinec ! Please Review and Merge !

**Pull request merged**